react-components/table-intro/step6.md

```diff
@@ -1,56 +1,106 @@
-As mentioned in step 4, we can handle the case for large datasets on the interaction side, also. We can do this by incorporating pagination controls that give users more granular control over their view of the data.
+In this step, we'll move our row/column definitions to an external file and add a few more rows of sample data to make the next part of the lesson more realistic.
 
-## Task
+1) <strong>Navigate to the `src` folder and open `src/data.js`.</strong>
 
-1) Import Pagination component from react-core right after your imports from  `@patternfly/react-table`.
+2) <strong>Copy the row/column definitions to the external file.</strong>
```

Comment:
Maybe be a little more specific here, like `Copy the row/column definitions from [file name] to the external file opened in step 1.

Reply:
I think the location is inferred from the previous steps.